src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -19,27 +18,60 @@ internal class MetadataProvider
 
         private readonly ConcurrentDictionary<FilePath, Watch<(ErrorList, UserMetadata)>> _metadataCache = new();
 
-        public ICollection<string> HtmlMetaHidden { get; }
-
-        public IReadOnlyDictionary<string, string> HtmlMetaNames { get; }
-
-        public MetadataProvider(Config config, Input input, BuildScope buildScope, JsonSchemaLoader jsonSchemaLoader)
+        public ICollection<string> HtmlMetaHidden { get; } = new List<string>()
```

Comment:
Move these two properties to `HtmlUtility.cs`, they are only used in `CreateHtmlMetaTags` method.

Reply:
Done.